src/auth/auth.proto

```diff
@@ -218,8 +218,13 @@ enum Permission {
   REPO_ADD_PIPELINE_READER    = 212;
   REPO_REMOVE_PIPELINE_READER = 213;
   REPO_ADD_PIPELINE_WRITER    = 214;
+  REPO_CREATE = 215;
```

Comment:
this should be PROJECT_CREATE_REPO if you want to be consistent with PROJECT_LIST_REPO below

Reply:
Hmm I prefer `REPO_CREATE` over `PROJECT_CREATE_REPO`, since since creating repos aren't tied to projects per se... what if I change `PROJECT_LIST_REPO` to just `LIST_REPO` or `REPO_LIST`?

Reply:
hmmm... 
`PROJECT_LIST_REPO` and `PROJECT_CREATE_REPO` seems to align more closely with our existing naming convention of specifying the most granular resource an action can be done against, and the action. For example, we have `REPO_LIST_FILE` which implies that for as granular as the REPO resource, we can do the action "LIST_FILE". 
WDYT?

Reply:
True, I understand that reasoning, but I was coming from the perspective of using the fewest words to convey the meaning of each permission, since I didn't want to add too much noise to these names. But I think this is largely subjective. I'll update to `PROJECT_*`